viziphant/statistics.py

```diff
@@ -41,3 +41,72 @@ def plot_isi(intervals, label, binsize=2*pq.ms, cutoff=250*pq.ms):
 
     return fig, ax
 
+
+def plot_patterns_statistics(patterns, winlen, binsize, n_neurons):
+    """
+    This function creates a histogram plot to visualise patterns statistics
+    output of a SPADE analysis.
+
+    Parameters
+    ----------
+    patterns : list
+        The output of elephant.spade
+    winlen : int
+        window length of the SPADE analysis
+    binsize : pq.Quantity
+        The bin size of the SPADE analysis
+    n_neurons : int
+        Number of neurons in the data set being analyzed
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+    ax : matplotlib.axes.Axes
+    """
+    patterns_dict = {'neurons': np.array([]),
+                     'occurrences': np.array([]),
+                     'pattern_size': np.array([]),
+                     'lags': np.array([])}
+    for pattern in patterns:
+        patterns_dict['neurons'] = \
+            np.append(patterns_dict['neurons'], pattern['neurons'])
```

Comment:
`np.append` copies the input array each time you append a value to it. Thus the complexity is quadratic. While this is probably OK to apply it here (I don't expect the `patterns` to be a large list), it should be avoided in general in favor of pythonic lists or, if the length is known, pre-allocated numpy arrays.
I'll take care of it here.

Reply:
all right, thanks!

Reply:
should I create then a separate script called `spade.py`?

Reply:
For now, keep it in statistics.py - it's not a big deal to move the function, once implemented, to another module.